docker-compose.yml

```diff
@@ -53,10 +53,10 @@ services:
     labels:
       - traefik.enable=true
       - traefik.docker.network=traefik
-      - traefik.http.routers.mailu.rule=Host(`mailu.${DOMAINNAME}`) || Host(`imap.${DOMAINNAME}`) || Host(`smtp.${DOMAINNAME}`)
+      - traefik.http.routers.mailu.rule=Host(`mailu.${DOMAINNAME}`)
```

Comment:
why? imap and smtp subdomains have to be routed to mailu

Reply:
Traefik only routes port 80 and 443. Imap and smtp use different ports.